### PR TITLE
Update 2021 09

### PR DESCRIPTION
**Pull request by rtgeha**
Half-yearly update. Still expecting additional input from ops

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -213,17 +213,17 @@
...
+        moved: -2
       },
       {
         quadrant: 2,
-        ring: 2,
+        ring: 0,
         label: "Lottie",
         active: false,
-        moved: 0
+        moved: 1
```
Comment (andreaseger) [inline docs/index.html:226]: isn't this a +2 move? or is only relevant if it's positive or negative?

File: docs/index.html [hunk 2]
```diff
@@ -312,22 +348,43 @@
...
       {
         quadrant: 1,
         ring: 0,
+        label: "Firebase",
+        active: false,
+        moved: 0
+      },
+      {
+        quadrant: 1,
+        ring: 3,
```
Comment (andreaseger) [inline docs/index.html]: Grafana put on `hold` is this intended?

File: docs/index.html [hunk 3]
```diff
@@ -523,10 +587,31 @@
...
+        quadrant: 0,
+        ring: 2,
+        label: "Kotlin Multiplatform",
+        active: true,
+        moved: 0
+      },
+      {
+        quadrant: 0,
+        ring: 2,
+        label: "Go",
```
Comment (andreaseger) [inline docs/index.html:604]: can we add a article for this describing for what it's used (AFAIK devops/CI things and the fastest-paths experiment)

File: docs/index.html [hunk 4]
```diff
@@ -523,10 +587,31 @@
...
         ring: 2,
-        label: "TypeScript",
+        label: "Kotlin Script",
+        active: true,
+        moved: 0
+      },
+      {
+        quadrant: 0,
+        ring: 2,
+        label: "Kotlin Multiplatform",
```
Comment (andreaseger) [inline docs/index.html:597]: is there some (wiki) page with information for what use-cases this is in "trial"

**Review by andreaseger — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (rtgeha) [reply docs/index.html:226]: Just positive/negative - it is not visualised on the radar how far it is moved.

File: docs/index.html (hunk 2 above)
Comment (rtgeha) [reply docs/index.html]: Thought we got the input from ops at the beginning of the review process, but I do not find where the input was given. Might be moved by mistake then

File: docs/index.html (hunk 4 above)
Comment (rtgeha) [reply docs/index.html:597]: It is in "assess". We got the input from dah that they want to have a look at it

Comment (andreaseger) [reply docs/index.html:597]: ah mixed up the rings. So just some early look at it.

File: docs/index.html (hunk 2 above)
Comment (rtgeha) [reply docs/index.html]: reverted the change to Grafana

**Review by andreaseger — APPROVED** [review]

File: docs/index.html (hunk 3 above)
Comment (rtgeha) [reply docs/index.html:604]: Added the link and created a preliminary page. Feel free to adapt.

**Review by ValentynKuznietsov — APPROVED** [review]

**Comment by ValentynKuznietsov** [comment]
no surprises for me

**Pull request merged**